CloudFoundry/content/apps/quotas.md

```diff
@@ -0,0 +1,114 @@
+---
+menu:
+  main:
+    parent: apps
+title: Quotas
+weight: -10
+---
+
+#### Quotas:
+
+Cloud Foundry capacity is managed by quota. Quotas provide a not to exceed reservation of memory, compute, application routes and service instances. 
+
+##### Quota allocations:
+
+We have allocated trial participants a default quota of 10GB of RAM, 1000 routes and 100 service instances.
+
+- 1000 application routes.
+- 10.024GB of RAM.
+- 100 service instances.
+
+###### Routes:
+
+Hostname and domain pairs where an application that exposes a listening port can be reached.
+
+###### RAM:
+
+The amount of RAM available to your applications.
+
+###### Service Instances:
+
+The number of service instances available to your organization. 
+
+###### Quota Limits:
+
+If a new application `push` would exceed your organization's quota the request will fail with status code `400` and a message describing the limit that would be exceeded.
+
+**Example:**
+
+	Creating app APPLICATION in org ORG / space SPACE as USER...
+	FAILED
+	Server error, status code: 400, error code: 100007, message: You have exceeded the memory limit for your organization's quota.
+
+In this situation you have three options:
+
+1. Delete existing resources with `cf delete`, `delete-service`, `delete-route` or similar.
+2. Reconfigure existing [Application Quotas](#application-quotas) and redeploy.
+3. Request a quota change [support@governmentpaas.zendesk.com](support@governmentpaas.zendesk.com).
+
+#### Application Quotas:
+
+As a Cloud Foundry user you’re free to divide your organization's quota capacity amongst your applications as you see fit by way of application quotas. Application limits are specified in your application manifest or as `cf push` command line options.
+
+###### Application Quota Options:
+
+`memory: / -m`
+
+The application memory limit. This setting has a dual-purpose as your application compute limit is derived from its memory limit. This relationship is explained in detail below.
+
+`disk_quota / -k`
+
+The maximum amount of disk space available to your app.
+
+`instances: / -i`
+
+Sets the number of application instances to launch. Each additional instance receives the same memory and disk reservation. An application with a manifest specifying `memory: 256M` and `instances: 4` would reserve 1GB (256M x 4) total.
+
+###### Application Quota Options : Memory Share equals Compute Share
+
+As noted above, your application's compute limit is derived from a its memory limit. Each application receives a compute share equal to its relative share of memory.
+
+For example, with a 1 unit (3.75GB) quota:
+
+- `memory: 1875`
+	- Guaranteed at least 50% vCPU time.
+	- Offered up to 100% vCPU time. 
+	- Limited to 1.875GB of RAM.
+- `memory: 375`
+	- Guaranteed at least 10% vCPU time. 
+	- Offered up to 100% vCPU time. 
+	- Limited to 375MB of RAM.
+
```

Comment:
I'm not 100% sure if this is how CPU allocation works. I think it depends on the ratio of the container memory to the actual DEA memory, which tenants might not know. The number of CPUs on DEA also impacts how much actual compute might a container get. This is also not known to the tenants....